Lesch

MartinLesch opened the pull request:
Java-Programme aus meiner Aufgabenstellung soweit überarbeitet und angepasst:
NICHT MEHR "Csv.java" (für alle Csv-Dateien) sondern jetzt eigene Klassen "CsvBuch" und "CsvKont". 
WUEB enthält Main, diese startet aber nur noch "guiLogin" übergibt beide vorgenannten Csv-Klassen.

VORSICHT!! NICHT EINFACH MERGEN!! Sind deletes für einige Programme dabei. Falls Löschung nicht erwünscht, die neuen Programme sind in dem Commit "Ein neuer Anfang".

martinlesch committed:
Delete buchungen.csv

martinlesch committed:
Delete skr03.csv

martinlesch committed:
Delete Anmeldung.java

martinlesch committed:
Delete Buchungen.java

martinlesch committed:
Delete Csv.java

martinlesch committed:
Delete Konten.java

martinlesch committed:
Delete Login.java

martinlesch committed:
Delete NeueBuchungen.java

martinlesch committed:
Update Operation.java

martinlesch committed:
Delete Register.java

martinlesch committed:
Delete TestClassMart.java

martinlesch committed:
Delete WUEB.java

martinlesch committed:
Delete guiBUCHUNG.form

martinlesch committed:
Delete guiBUCHUNG.java

martinlesch committed:
Delete guiLOGIN.form

martinlesch committed:
Delete guiLOGIN.java

martinlesch committed:
Delete guiWUEB.form

martinlesch committed:
Delete guiWUEB.java

martinlesch committed:
Delete wueb_uml.cdg

martinlesch committed:
Ein neuer Anfang ... (lesch)

CSV-Handling nun in 2 getrennten Klassen (CsvKont und CsvBuch). Dateien werden im gleichen Verzeichnis wie die Java-Source-Codes gesucht/gespeichert, falls hiervon abweichend Package verwendet wird, muss dies angepasst werden im Pfad.
Die Klasse WUEB enthält die Main Klasse, die guiLogin startet und als Übergabewert je ein Objekt "CsvBuch" und "CsvKont" überträgt. Müsste dann im Konstuktor der guiLogin entsprechend entgegen genommen werden und an die guiWueb weitergegeben werden.

BlackCircle22 reviewed (commented):
package wueb added, kann so übernommen werden

martinlesch committed:
09.10.2017 div. kleine Änderungen 

skr03.csv --> Speicherformat UTF-8 (Umlaute)
skr03.csv --> ohne Ueberschriften

buch.csv --> Inhalt (Testdaten) reduziert 

ReadWriteDES --> Fehler via try/catch abgefangen nicht mehr "throw"

Operation --> Nettoberechnung ueberarbeitet

guiWUEB --> Formatierung der Zahlen (deutsch)

csvBuch --> Mehrwertsteuer mit 7% abgefragt

martinlesch committed:
Delete Buch.csv

martinlesch committed:
Delete CsvBuch.java

martinlesch committed:
Delete Kontoart.java

martinlesch committed:
Delete Operation.java

martinlesch committed:
Delete ReadWriteDES.java

martinlesch committed:
Delete Wueb.java

martinlesch committed:
Delete guiWUEB.form

martinlesch committed:
Delete guiWUEB.java

martinlesch committed:
Delete skr03.csv

martinlesch committed:
09.10.2017 div. kleine Änderungen

skr03.csv --> Speicherformat UTF-8 (Umlaute)
skr03.csv --> ohne Ueberschriften

buch.csv --> Inhalt (Testdaten) reduziert 

ReadWriteDES --> Fehler via try/catch abgefangen nicht mehr "throw"

Operation --> Nettoberechnung ueberarbeitet

guiWUEB --> Formatierung der Zahlen (deutsch)

csvBuch --> Mehrwertsteuer mit 7% abgefragt
